src/cpp/processquit.cpp

```diff
@@ -74,8 +74,8 @@ class SafeSocketNotifier : public QObject
 public slots:
 	void setEnabled(bool enable)       { sn->setEnabled(enable); }
 
-signals:
-	void activated(int socket);
+public:
+	SignalInt activated;
```

Comment:
This change won't work since the connect call's destination is a Qt signal. To make it work, connect to a Qt slot that calls the boost signal.

Reply:
I see makes senes! updating

Reply:
after another look, i realized the connect destination is in fact a slot. can you please clarify your concern.

Reply:
Line 61 specifies a signal as the destination.

Reply:
got it, updated, thanks